Resfeber/Views/TripCell.swift

```diff
@@ -80,7 +80,10 @@ private extension TripCell {
 
     func updateViews() {
         if let trip = trip {
-            imageView.image = nil
+            if let data = trip.image {
```

Comment:
Would it be better to rename the `Trip` attribute from `image` to `imageData` and then extend `Trip` to contain a `UIImage` computed variable, like this:

```
extension Trip {
    var image: UIImage? {
        UIImage(data: imageData)
    }
}
```